CHANGELOG.md

```diff
@@ -6,31 +6,53 @@
+- The `stellargraph.data.load_dataset_BlogCatalog3` function has been replaced by the `load` method on `stellargraph.datasets.BlogCatalog3` [\#888](http://github.com/stellargraph/stellargraph/pull/888). Migration: replace `load_dataset_BlogCatalog3(location)` with `BlogCatalog3().load()`; code required to find the location or download the dataset can be removed, as `load` now does this automatically.
 - `stellargraph.data.train_test_val_split` and `stellargraph.data.NodeSplitter` have been removed. [\#887](https://github.com/stellargraph/stellargraph/pull/887) Migration: this functionality should be replaced with `pandas` and `sklearn` (for instance, `sklearn.model_selection.train_test_split`).
 
 ### Experimental features
 
 Some new algorithms and features are still under active development, and are available as an experimental preview. However, they may not be easy to use: their documentation or testing may be incomplete, and they may change dramatically from release to release. The experimental status is noted in the documentation and at runtime via prominent warnings.
 
+- Temporal Random Walks: random walks that respect the time that each edge occurred (stored as edge weights) [\#787](http://github.com/stellargraph/stellargraph/pull/787). The implementation does not have an example or thorough testing and documentation.
 - Watch Your Step: computes node embeddings by simulating the effect of random walks, rather than doing them. [\#750](https://github.com/stellargraph/stellargraph/pull/750). The implementation is not fully tested.
+- GraphWave: computes structural node embeddings by using wavelet transforms on the graph Laplacian [\#814](https://github.com/stellargraph/stellargraph/pull/814), [\#822](https://github.com/stellargraph/stellargraph/issues/822). The implementation is `O(n^3)`, where `n` is the number of nodes.
 - ComplEx: computes embeddings for nodes and edge types in knowledge graphs, and use these to perform link prediction [\#756](https://github.com/stellargraph/stellargraph/issues/756). The implementation hasn't been validated to match the paper.
+- Neo4j connector: the GraphSAGE algorithm can execute doing neighbourhood sampling in a Neo4j database, so that the edges of a graph do not have to fit entirely into memory [\#799](http://github.com/stellargraph/stellargraph/pull/799). The implementation is not automatically tested, and doesn't support functionality like loading node feature vectors from Neo4j.
 
 ### Bug fixes and other changes
 
-- StellarGraph now supports [`tensorflow 2.1`](https://github.com/tensorflow/tensorflow/releases/tag/v2.1.0), which includes GPU support by default: [\#875](https://github.com/stellargraph/stellargraph/pull/875)
+- StellarGraph now supports [`tensorflow` 2.1](https://github.com/tensorflow/tensorflow/releases/tag/v2.1.0), which includes GPU support by default: [\#875](https://github.com/stellargraph/stellargraph/pull/875)
```

Comment:
my formatting here was intentionally since I think Github's rendering looks ugly, what do you think?

Reply:
Ah, that's true. Since this is referring to TensorFlow in text rather than as part of a piece of code, I'm going to switch to no backticks at all, that is:

```suggestion
- StellarGraph now supports [TensorFlow 2.1](https://github.com/tensorflow/tensorflow/releases/tag/v2.1.0), which includes GPU support by default: [\#875](https://github.com/stellargraph/stellargraph/pull/875)
```